src/cloud-api-adaptor/Makefile

```diff
@@ -172,12 +172,21 @@ endif
 REGISTRY ?= quay.io/confidential-containers
 
 PODVM_DISTRO ?= ubuntu
+ifeq ($(PODVM_SHORT_TAG),)
```

Comment:
@ifireball this has a typo in the SoB line..  
Note the suffix a: `Barak Korren <bkorren@redhat.com>a`